modules/claims_api/spec/lib/claims_api/local_bgs_spec.rb

```diff
@@ -40,18 +40,6 @@
         ClaimsApi::LocalBGS.breakers_service.end_forced_outage!
       end
     end
-
-    it 'triggers StatsD measurements' do
-      VCR.use_cassette('claims_api/bgs/claimant_web_service/find_poa_by_participant_id',
-                       allow_playback_repeats: true) do
-        allow_any_instance_of(BGS::OrgWebService).to receive(:find_poa_history_by_ptcpnt_id).and_return({})
-
-        %w[establish_ssl_connection connection_wsdl_get connection_post parsed_response].each do |event|
-          expect { subject.find_poa_by_participant_id('does-not-matter') }
-            .to trigger_statsd_measure("api.claims_api.local_bgs.#{event}.duration")
-        end
-      end
-    end
```

Comment:
It doesn't seem horrible to assert about this component that it is emitting the correct `statsd`  entries. Why is this being removed?

Reply:
I think we could achieve re-establishing this test if we just remove the `connection_wsdl_get`